bcda/client/bluebutton_test.go

```diff
@@ -20,8 +20,9 @@ type BBTestSuite struct {
 
 func (s *BBTestSuite) SetupTest() {
 	s.ts = httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		w.Header().Set("Content-Type", "application/fhir+json")
-		fmt.Fprint(w, `{ "test": "ok" }`)
+		w.Header().Set("Content-Type", r.URL.Query().Get("_format"))
+		response := fmt.Sprintf("{ \"test\": \"ok\"; \"url\": %v}", r.URL.String())
```

Comment:
ohhhh so does the printed out url contain the added "excludeSAMHSA=true" now?

Reply:
sort of.  The BB client makes a request to this TLSServer instead of actual blue button.  All I did was rewrite the response to include the url that the client made the request to.  This lets me determine exactly what params were added to the request.